### PR TITLE
ADEN-12878 | Add amazonVideo.sample.json

### PR DESCRIPTION
**Pull request by rafalgiemza**
https://fandom.atlassian.net/browse/ADEN-12878

### CODE REVIEW

File: line-item-presets/amazonVideo.sample.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  "priceMap": "v_1e728sg,v_qw0cn4,v_1yq8740,v_30z9q8,v_14rg1s0,v_jebbb4,v_1ogn7y8,v_gexog0,v_1bxwjk0,v_weja4g,v_1qdbfgg,v_8ji77k,v_12iacjk,v_myx340,v_1ivme4g,v_9dgzcw,v_1h7otts,v_twmxog,v_1vn3pq8,v_16sykg,v_1694v7k,v_lpyww0,v_1ngfoxs,v_dyaa68,v_19uzl6o,v_xnhgcg,v_1ue5ji8,v_6gl8u8,v_10fde68,v_pob4lc,v_1jn39xc,v_cwtszk,v_1enal1c,v_rc8ow0,v_1ychr7k,v_2n8tts,v_14dplvk,v_jujnk0,v_1owvk74,v_gnobnk,v_1amggzk,v_v337k0,v_1rtraps,v_7824n4,v_134rfnk,v_nle680,v_1hk6bk0,v_hqdr0g,v_1bp5wcg,v_w5smww,v_1rdiygw,v_atwum8,v_u8hybk,v_9pszk,v_ksvrb4,v_ez4a9s,v_zia8lc,v_5ji39c,v_ojq9z4,v_bs8ydc,v_s5kzy8,v_4ajx1c,v_ipysxs,v_fql62o,v_wxvzsw,v_88w4qo,v_mob0n4,v_9wtp1c,v_sx1vr4,v_1q5o8w,v_lfcuf4,v_6rtse8,v_13ihvk0,v_nz4m4g,v_1hxwrgg,v_dno7pc,v_5s8qgw,v_cpyl1c,v_3ace0w,v_harvuo,v_716wow,v_a5kc8w,v_py58g,v_ffcmio,v_4gsnwg,v_adoidc,v_1fg0f0g,v_uwugow,v_1wnb8qo,v_beiigw,v_3kn7y8,v_gc4jk0,v_8ufi80,v_8zfaww,v_1hq9kw,v_e97l6o,v_6a19fk,v_d7r400,v_2975ds,v_y2bcw,v_17gu39c,v_k5s740,v_1lw8z5s,v_gylmo0,v_f5zi80,v_194rnk0,v_yv6oe8,v_1stytq8,v_4wej28,v_11n2m80,v_pfkhds,v_1knasxs,v_b55e68,v_1cvm680,v_scg7wg,v_1xes4jk,v_3ngcu8,v_15dx4w0,v_i2v8qo",
+  "lineItemName": "A9 featured video N&R %%element%%",
```
Comment (dmnsobczak) [inline line-item-presets/amazonVideo.sample.json]:
```suggestion
  "lineItemName": "News&Ratings Video - $%%element%%",
```

File: line-item-presets/amazonVideo.sample.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+  "orderId": "3170606300",
```
Comment (dmnsobczak) [inline line-item-presets/amazonVideo.sample.json:2]:
```suggestion
  "orderId": "3167997490",
```

File: line-item-presets/amazonVideo.sample.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  "creativeId": "138426662160",
```
Comment (dmnsobczak) [inline line-item-presets/amazonVideo.sample.json:3]:
```suggestion
  "creativeId": "138426329086",
```

File: line-item-presets/amazonVideo.sample.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  "iterator": "0.25,0.50,0.75,1.00,1.25,1.50,1.75,10.00,10.25,10.50,10.75,11.00,11.25,11.50,11.75,12.00,12.25,12.50,12.75,13.00,13.25,13.50,13.75,14.00,14.25,14.50,14.75,15.00,15.25,15.50,15.75,16.00,16.25,16.50,16.75,17.00,17.25,17.50,17.75,18.00,18.25,18.50,18.75,19.00,19.25,19.50,19.75,2.00,2.25,2.50,2.75,20.00,20.50,21.00,21.50,22.00,22.50,23.00,23.50,24.00,24.50,25.00,25.50,26.00,26.50,27.00,27.50,28.00,28.50,29.00,29.50,3.00,3.25,3.50,3.75,30.00,31.00,32.00,33.00,34.00,35.00,36.00,37.00,38.00,39.00,4.00,4.25,4.50,4.75,40.00,41.00,42.00,43.00,44.00,45.00,46.00,47.00,48.00,49.00,5.00,5.25,5.50,5.75,50.00,6.00,6.25,6.50,6.75,7.00,7.25,7.50,7.75,8.00,8.25,8.50,8.75,9.00,9.25,9.50",
```
Comment (dmnsobczak) [inline line-item-presets/amazonVideo.sample.json]: The prices are not sorted - are you 100% they match the A9 price map? Can you sort it?

File: line-item-presets/amazonVideo.sample.json [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+  "lineItemName": "A9 featured video N&R %%element%%",
+  "sizes": "640x480",
```
Comment (dmnsobczak) [inline line-item-presets/amazonVideo.sample.json:7]:
```suggestion
  "sizes": "640x480",
  "isVideo": true,
```

File: line-item-presets/amazonVideo.sample.json [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+  "sameAdvertiser": true,
+  "type": "PRICE_PRIORITY",
+  "priority": "10",
+  "start": "",
+  "end": "",
+  "rate": "%%element%%",
+  "adUnit": "",
+  "keys": [
+    "amznbid",
+    "amznsz"
```
Comment (dmnsobczak) [inline line-item-presets/amazonVideo.sample.json]:
```suggestion
    "src"
```

File: line-item-presets/amazonVideo.sample.json [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+  ],
+  "operators": [
+    "IS",
+    "IS"
+  ],
+  "values": [
+    "%%priceMapElement%%",
+    "640x480"
```
Comment (dmnsobczak) [inline line-item-presets/amazonVideo.sample.json]:
```suggestion
    "comicvine,gamefaqs,gamespot,giantbomb,metacritic,tvguide"
```

File: line-item-presets/amazonVideo.sample.json [hunk 8]
```diff
@@ -0,0 +1,28 @@
+{
+  "orderId": "3167997490",
+  "creativeId": "138426329086",
+  "iterator": "0.25,0.50,0.75,1.00,1.25,1.50,1.75,10.00,10.25,10.50,10.75,11.00,11.25,11.50,11.75,12.00,12.25,12.50,12.75,13.00,13.25,13.50,13.75,14.00,14.25,14.50,14.75,15.00,15.25,15.50,15.75,16.00,16.25,16.50,16.75,17.00,17.25,17.50,17.75,18.00,18.25,18.50,18.75,19.00,19.25,19.50,19.75,2.00,2.25,2.50,2.75,20.00,20.50,21.00,21.50,22.00,22.50,23.00,23.50,24.00,24.50,25.00,25.50,26.00,26.50,27.00,27.50,28.00,28.50,29.00,29.50,3.00,3.25,3.50,3.75,30.00,31.00,32.00,33.00,34.00,35.00,36.00,37.00,38.00,39.00,4.00,4.25,4.50,4.75,40.00,41.00,42.00,43.00,44.00,45.00,46.00,47.00,48.00,49.00,5.00,5.25,5.50,5.75,50.00,6.00,6.25,6.50,6.75,7.00,7.25,7.50,7.75,8.00,8.25,8.50,8.75,9.00,9.25,9.50",
```
Comment (rafalgiemza) [inline line-item-presets/amazonVideo.sample.json]:
```suggestion
  "iterator": "0.25,0.50,0.75,1.00,1.25,1.50,1.75,2.00,2.25,2.50,2.75,3.00,3.25,3.50,3.75,4.00,4.25,4.50,4.75,5.00,5.25,5.50,5.75,6.00,6.25,6.50,6.75,7.00,7.25,7.50,7.75,8.00,8.25,8.50,8.75,9.00,9.25,9.50,10.00,10.25,10.50,10.75,11.00,11.25,11.50,11.75,12.00,12.25,12.50,12.75,13.00,13.25,13.50,13.75,14.00,14.25,14.50,14.75,15.00,15.25,15.50,15.75,16.00,16.25,16.50,16.75,17.00,17.25,17.50,17.75,18.00,18.25,18.50,18.75,19.00,19.25,19.50,19.75,20.00,20.50,21.00,21.50,22.00,22.50,23.00,23.50,24.00,24.50,25.00,25.50,26.00,26.50,27.00,27.50,28.00,28.50,29.00,29.50,30.00,31.00,32.00,33.00,34.00,35.00,36.00,37.00,38.00,39.00,40.00,41.00,42.00,43.00,44.00,45.00,46.00,47.00,48.00,49.00,50.00,",
```

File: line-item-presets/amazonVideo.sample.json [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+  "priceMap": "v_1e728sg,v_qw0cn4,v_1yq8740,v_30z9q8,v_14rg1s0,v_jebbb4,v_1ogn7y8,v_gexog0,v_1bxwjk0,v_weja4g,v_1qdbfgg,v_8ji77k,v_12iacjk,v_myx340,v_1ivme4g,v_9dgzcw,v_1h7otts,v_twmxog,v_1vn3pq8,v_16sykg,v_1694v7k,v_lpyww0,v_1ngfoxs,v_dyaa68,v_19uzl6o,v_xnhgcg,v_1ue5ji8,v_6gl8u8,v_10fde68,v_pob4lc,v_1jn39xc,v_cwtszk,v_1enal1c,v_rc8ow0,v_1ychr7k,v_2n8tts,v_14dplvk,v_jujnk0,v_1owvk74,v_gnobnk,v_1amggzk,v_v337k0,v_1rtraps,v_7824n4,v_134rfnk,v_nle680,v_1hk6bk0,v_hqdr0g,v_1bp5wcg,v_w5smww,v_1rdiygw,v_atwum8,v_u8hybk,v_9pszk,v_ksvrb4,v_ez4a9s,v_zia8lc,v_5ji39c,v_ojq9z4,v_bs8ydc,v_s5kzy8,v_4ajx1c,v_ipysxs,v_fql62o,v_wxvzsw,v_88w4qo,v_mob0n4,v_9wtp1c,v_sx1vr4,v_1q5o8w,v_lfcuf4,v_6rtse8,v_13ihvk0,v_nz4m4g,v_1hxwrgg,v_dno7pc,v_5s8qgw,v_cpyl1c,v_3ace0w,v_harvuo,v_716wow,v_a5kc8w,v_py58g,v_ffcmio,v_4gsnwg,v_adoidc,v_1fg0f0g,v_uwugow,v_1wnb8qo,v_beiigw,v_3kn7y8,v_gc4jk0,v_8ufi80,v_8zfaww,v_1hq9kw,v_e97l6o,v_6a19fk,v_d7r400,v_2975ds,v_y2bcw,v_17gu39c,v_k5s740,v_1lw8z5s,v_gylmo0,v_f5zi80,v_194rnk0,v_yv6oe8,v_1stytq8,v_4wej28,v_11n2m80,v_pfkhds,v_1knasxs,v_b55e68,v_1cvm680,v_scg7wg,v_1xes4jk,v_3ngcu8,v_15dx4w0,v_i2v8qo",
```
Comment (rafalgiemza) [inline line-item-presets/amazonVideo.sample.json]:
```suggestion
  "priceMap": "v_1e728sg,v_qw0cn4,v_1yq8740,v_30z9q8,v_14rg1s0,v_jebbb4,v_1ogn7y8,v_hqdr0g,v_1bp5wcg,v_w5smww,v_1rdiygw,v_6rtse8,v_13ihvk0,v_nz4m4g,v_1hxwrgg,v_adoidc,v_1fg0f0g,v_uwugow,v_1wnb8qo,v_y2bcw,v_17gu39c,v_k5s740,v_1lw8z5s,v_f5zi80,v_194rnk0,v_yv6oe8,v_1stytq8,v_4wej28,v_11n2m80,v_pfkhds,v_1knasxs,v_b55e68,v_1cvm680,v_scg7wg,v_1xes4jk,v_3ngcu8,v_15dx4w0,v_i2v8qo,v_gexog0,v_1bxwjk0,v_weja4g,v_1qdbfgg,v_8ji77k,v_12iacjk,v_myx340,v_1ivme4g,v_9dgzcw,v_1h7otts,v_twmxog,v_1vn3pq8,v_16sykg,v_1694v7k,v_lpyww0,v_1ngfoxs,v_dyaa68,v_19uzl6o,v_xnhgcg,v_1ue5ji8,v_6gl8u8,v_10fde68,v_pob4lc,v_1jn39xc,v_cwtszk,v_1enal1c,v_rc8ow0,v_1ychr7k,v_2n8tts,v_14dplvk,v_jujnk0,v_1owvk74,v_gnobnk,v_1amggzk,v_v337k0,v_1rtraps,v_7824n4,v_134rfnk,v_nle680,v_1hk6bk0,v_atwum8,v_u8hybk,v_9pszk,v_ksvrb4,v_ez4a9s,v_zia8lc,v_5ji39c,v_ojq9z4,v_bs8ydc,v_s5kzy8,v_4ajx1c,v_ipysxs,v_fql62o,v_wxvzsw,v_88w4qo,v_mob0n4,v_9wtp1c,v_sx1vr4,v_1q5o8w,v_lfcuf4,v_dno7pc,v_5s8qgw,v_cpyl1c,v_3ace0w,v_harvuo,v_716wow,v_a5kc8w,v_py58g,v_ffcmio,v_4gsnwg,v_beiigw,v_3kn7y8,v_gc4jk0,v_8ufi80,v_8zfaww,v_1hq9kw,v_e97l6o,v_6a19fk,v_d7r400,v_2975ds,v_gylmo0",
```

**Review by rafalgiemza — COMMENTED** [review]
iterator & priceMap is now sorted

File: line-item-presets/amazonVideo.sample.json [hunk 10]
```diff
@@ -0,0 +1,28 @@
+{
+  "orderId": "3167997490",
+  "creativeId": "138426329086",
+  "iterator": "0.25,0.50,0.75,1.00,1.25,1.50,1.75,2.00,2.25,2.50,2.75,3.00,3.25,3.50,3.75,4.00,4.25,4.50,4.75,5.00,5.25,5.50,5.75,6.00,6.25,6.50,6.75,7.00,7.25,7.50,7.75,8.00,8.25,8.50,8.75,9.00,9.25,9.50,10.00,10.25,10.50,10.75,11.00,11.25,11.50,11.75,12.00,12.25,12.50,12.75,13.00,13.25,13.50,13.75,14.00,14.25,14.50,14.75,15.00,15.25,15.50,15.75,16.00,16.25,16.50,16.75,17.00,17.25,17.50,17.75,18.00,18.25,18.50,18.75,19.00,19.25,19.50,19.75,20.00,20.50,21.00,21.50,22.00,22.50,23.00,23.50,24.00,24.50,25.00,25.50,26.00,26.50,27.00,27.50,28.00,28.50,29.00,29.50,30.00,31.00,32.00,33.00,34.00,35.00,36.00,37.00,38.00,39.00,40.00,41.00,42.00,43.00,44.00,45.00,46.00,47.00,48.00,49.00,50.00,",
```
Comment (rafalgiemza) [inline line-item-presets/amazonVideo.sample.json]:
```suggestion
  "iterator": "0.25,0.50,0.75,1.00,1.25,1.50,1.75,2.00,2.25,2.50,2.75,3.00,3.25,3.50,3.75,4.00,4.25,4.50,4.75,5.00,5.25,5.50,5.75,6.00,6.25,6.50,6.75,7.00,7.25,7.50,7.75,8.00,8.25,8.50,8.75,9.00,9.25,9.50,10.00,10.25,10.50,10.75,11.00,11.25,11.50,11.75,12.00,12.25,12.50,12.75,13.00,13.25,13.50,13.75,14.00,14.25,14.50,14.75,15.00,15.25,15.50,15.75,16.00,16.25,16.50,16.75,17.00,17.25,17.50,17.75,18.00,18.25,18.50,18.75,19.00,19.25,19.50,19.75,20.00,20.50,21.00,21.50,22.00,22.50,23.00,23.50,24.00,24.50,25.00,25.50,26.00,26.50,27.00,27.50,28.00,28.50,29.00,29.50,30.00,31.00,32.00,33.00,34.00,35.00,36.00,37.00,38.00,39.00,40.00,41.00,42.00,43.00,44.00,45.00,46.00,47.00,48.00,49.00,50.00",
```

File: line-item-presets/amazonVideo.sample.json [hunk 11]
```diff
@@ -0,0 +1,28 @@
+{
+  "orderId": "3167997490",
```
Comment (dmnsobczak) [inline line-item-presets/amazonVideo.sample.json]:
```suggestion
  "orderId": "3170606300",
```

File: line-item-presets/amazonVideo.sample.json [hunk 12]
```diff
@@ -0,0 +1,28 @@
...
+  "creativeId": "138426329086",
```
Comment (dmnsobczak) [inline line-item-presets/amazonVideo.sample.json]:
```suggestion
  "creativeId": "138426662160",
```

File: line-item-presets/amazonVideo.sample.json [hunk 13]
```diff
@@ -0,0 +1,28 @@
...
+  "priceMap": "v_1e728sg,v_qw0cn4,v_1yq8740,v_30z9q8,v_14rg1s0,v_jebbb4,v_1ogn7y8,v_hqdr0g,v_1bp5wcg,v_w5smww,v_1rdiygw,v_6rtse8,v_13ihvk0,v_nz4m4g,v_1hxwrgg,v_adoidc,v_1fg0f0g,v_uwugow,v_1wnb8qo,v_y2bcw,v_17gu39c,v_k5s740,v_1lw8z5s,v_f5zi80,v_194rnk0,v_yv6oe8,v_1stytq8,v_4wej28,v_11n2m80,v_pfkhds,v_1knasxs,v_b55e68,v_1cvm680,v_scg7wg,v_1xes4jk,v_3ngcu8,v_15dx4w0,v_i2v8qo,v_gexog0,v_1bxwjk0,v_weja4g,v_1qdbfgg,v_8ji77k,v_12iacjk,v_myx340,v_1ivme4g,v_9dgzcw,v_1h7otts,v_twmxog,v_1vn3pq8,v_16sykg,v_1694v7k,v_lpyww0,v_1ngfoxs,v_dyaa68,v_19uzl6o,v_xnhgcg,v_1ue5ji8,v_6gl8u8,v_10fde68,v_pob4lc,v_1jn39xc,v_cwtszk,v_1enal1c,v_rc8ow0,v_1ychr7k,v_2n8tts,v_14dplvk,v_jujnk0,v_1owvk74,v_gnobnk,v_1amggzk,v_v337k0,v_1rtraps,v_7824n4,v_134rfnk,v_nle680,v_1hk6bk0,v_atwum8,v_u8hybk,v_9pszk,v_ksvrb4,v_ez4a9s,v_zia8lc,v_5ji39c,v_ojq9z4,v_bs8ydc,v_s5kzy8,v_4ajx1c,v_ipysxs,v_fql62o,v_wxvzsw,v_88w4qo,v_mob0n4,v_9wtp1c,v_sx1vr4,v_1q5o8w,v_lfcuf4,v_dno7pc,v_5s8qgw,v_cpyl1c,v_3ace0w,v_harvuo,v_716wow,v_a5kc8w,v_py58g,v_ffcmio,v_4gsnwg,v_beiigw,v_3kn7y8,v_gc4jk0,v_8ufi80,v_8zfaww,v_1hq9kw,v_e97l6o,v_6a19fk,v_d7r400,v_2975ds,v_gylmo0",
+  "lineItemName": "News&Ratings Video - $%%element%%",
```
Comment (dmnsobczak) [inline line-item-presets/amazonVideo.sample.json]:
```suggestion
  "lineItemName": "News&Ratings A9 Video - $%%element%%",
```

File: line-item-presets/amazonVideo.sample.json (hunk 4 above)
Comment (rafalgiemza) [reply line-item-presets/amazonVideo.sample.json]: fixed

**Review by dmnsobczak — APPROVED** [review]

**Pull request merged**